cmd/poller/plugin/README.md

```diff
@@ -120,6 +120,9 @@ Notice that the rules are executed in the same order as you've added them. List
   - [exclude_equals](#exclude_equals)
   - [exclude_contains](#exclude_contains)
   - [exclude_regex](#exclude_regex)
+  - [include_equals](#include_equals)
```

Comment:
We should add in readme that include and exclude should not apply on the same label OR order matters like below. 
As exclude applicable for given record, but include would change for whole records.

Example:
exclude_equals: style `flexgroup_constituent`.  [flexvol and flexgroup comes in result]
include_equals: style `flexgroup`.                        [flexvols removed]


include_equals: style `flexgroup`.                        [only flexgroup comes in result]
exclude_equals: style `flexgroup_constituent`.  [No effect]

Reply:
Not sure if we should document such cases as it will be implicit by rule defintion in docs.